pandas/_testing.py

```diff
@@ -956,6 +965,8 @@ def assert_numpy_array_equal(
     obj : str, default 'numpy array'
         Specify object name being compared, internally used to show appropriate
         assertion message.
+    index : numpy.ndarray
```

Comment:
Could you specify the default value (None) here?

Reply:
Yes - I missed that.  Will fix.